docs/docs/multiplayer/player-info.md

```diff
@@ -16,6 +16,14 @@ Here’s the values for each player:
 - `avatarUrl: string`
 - `playerId: string` (same as key, just provided for simplicity)
 
+## Avatars
+
+Using avatars is a great way to personalize the UI to show who's turn it is or in a leaderboard. Since the avatar is loaded over the network there might be a slight delay during which you might want to display a placeholder – and we got you covered!
```

Comment:
Should this be `whose` instead of `who's`?

Reply:
Nope, it's short for `who is`

Reply:
But then you are saying `to show who is turn it is` instead of `to show whose turn it is`. That sounds off.

Reply:
You're right! Fixed in https://github.com/rune/rune-games-sdk/commit/bd7c4475b342c7d84c633fa5001a4e5555555ae3